### DISCUSSION TITLE
CUTE 知乎专栏: https://www.zhihu.com/people/reed-84-49/posts

### DISCUSSION THREAD

**Discussion by hwu36** [NVIDIA/cutlass#1252] [General]
@reed-lau , yours?

**Comment by MARD1NO** [comment]
> @reed-lau , yours?

I guess it is, reed's zhihu post really help me a lot. Maybe cutlass team can work with reed to publish some tutorials? :D

**Comment by reed-lau** [comment]
> @reed-lau , yours?

Hi @hwu36, I apologize for the delayed response. I created the cute series as my personal study notes and to assist others in learning about cute. Additionally, I might include some basic information to make the learning process smoother. If I can help, please let me know.

**Comment by hwu36** [comment]
I think @ziyuhuang123 also has a wechat group about cute

**Reply by MARD1NO** [reply]
Maybe add a wechat group QR code here or in README.md?

**Reply by xws117** [reply]
so，is there anyone could pull me in the wechat group ,my wechat is shamy117

**Reply by zhang662817** [reply]
@hwu36 @ziyuhuang123 can you also invite me to wechat group about cute? wechat: zhang662817

**Reply by ziyuhuang123** [reply]
OK bros! I will do it now!

**Reply by jeejeelee** [reply]
@ziyuhuang123 ，can you share wechat group QR code？

**Reply by guohaoqiang** [reply]
anyone can share the qr code please?

**Reply by mutou-wl** [reply]
can you also invite me to wechat group about cute? vx: wcp-wl

**Reply by HydraQYH** [reply]
can you also invite me to wechat group about cute? vx:Qyh18514037007

**Reply by MARD1NO** [reply]
can you also invite me to wechat group about cute? thanks! vx:i_am_kkz @ziyuhuang123

**Reply by luliyucoordinate** [reply]
can you also invite me to wechat group about cute? vx: coordinate_2020

**Reply by Daydream0929** [reply]
can you also invite me to wechat group about cute? vx: fengzhenjun0929

**Reply by defei-coder** [reply]
@ziyuhuang123 can you also invite me to wechat group about cute? vx: wxid_dcgqpnmuq60n21

**Reply by shenzhenghai** [reply]
@ziyuhuang123 could you please add me to the wechat group? thanks. 
vx: shen_zheng_hai

**Reply by xziya** [reply]
@ziyuhuang123 Please help, thanks 😄. vx: weizixm_

**Reply by bluebabyxp** [reply]
@ziyuhuang123 could you please add me to the wechat group? thanks. vx: bluebabyxp

**Reply by zebnut** [reply]
@ziyuhuang123 could you please add me to the wechat group? thanks. vx: wxtoujours

**Reply by justpi** [reply]
@ziyuhuang123 could you please add me to the wechat group? thanks. vx: leegaojun

**Reply by jpli02** [reply]
@zixuanweeei could you please add me to the wechat group? thanks: vs: li5560202

**Reply by haozhihan** [reply]
@zixuanweeei could you please add me to the wechat group? thanks: vs: north_hhz

**Reply by captain2017** [reply]
@ziyuhuang123 could you please add me to the wechat group? thanks. vx: 13073657821

**Reply by liuqi123123** [reply]
@ziyuhuang123 could you please add me to the wechat group? thanks. (vx:dlow67)

**Reply by tlogn** [reply]
@ziyuhuang123 could you please add me to the wechat group? thanks. vx: wxid_u1d6p8ivg7ib22

**Reply by xiefan46** [reply]
@ziyuhuang123 please also add me to the group. Thanks! vx: jiayoufan66

**Reply by HarryWu99** [reply]
@ziyuhuang123 please also add me to the group. Thanks! vx: harrywu67

**Reply by kefan203** [reply]
@ziyuhuang123 could you please add me to the wechat group? thanks. vx: ckf726

**Reply by Shan19900305** [reply]
@ziyuhuang123 could you please add me to the wechat group? thanks. vx: benjamin199003
